ST662_Group6.ipynb

```diff
@@ -2,40 +2,600 @@
+       "\n",
+       "  Company response to consumer Timely response?  Consumer disputed?  \\\n",
+       "0      Closed with explanation               No                 NaN   \n",
+       "1      Closed with explanation              Yes                 NaN   \n",
+       "2      Closed with explanation              Yes                 NaN   \n",
+       "3      Closed with explanation              Yes                 NaN   \n",
+       "4      Closed with explanation              Yes                 NaN   \n",
+       "\n",
+       "   Complaint ID  \n",
+       "0       4785819  \n",
+       "1       4096955  \n",
+       "2       3929916  \n",
+       "3       4251897  \n",
+       "4       4616090  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "#start running from here (as this file has the columns we need plus more)\n",
-    "df_Group6 = pd.read_csv('complaintsmaster.csv')"
+    "df_Group6 = pd.read_csv('complaintsmaster.csv')\n",
+    "df_Group6.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array(['OH', 'TN', 'FL', 'PA', 'CA', 'TX', 'VA', 'MO', 'GA', 'NY', 'CO',\n",
+       "       'WA', 'MN', 'AL', 'IL', 'NC', 'OK', 'MD', 'MI', 'LA', 'SC', nan,\n",
+       "       'OR', 'NH', 'IN', 'NJ', 'DC', 'UT', 'KY', 'NV', 'MA', 'AA', 'RI',\n",
+       "       'KS', 'DE', 'WI', 'MS', 'NM', 'PR', 'NE', 'CT', 'AR', 'AZ', 'IA',\n",
+       "       'ME', 'HI', 'WV', 'ID', 'ND', 'MT',\n",
+       "       'UNITED STATES MINOR OUTLYING ISLANDS', 'AE', 'WY', 'GU', 'SD',\n",
+       "       'AK', 'AP', 'VI', 'VT', 'MP', 'AS', 'MH'], dtype=object)"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Unique values of States Column\n",
+    "\n",
+    "states_list = df_Group6['State']\n",
+    "states_list.unique()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Diccionary of State initials and Full Name\n",
+    "\n",
+    "state_names = {\n",
+    "    'OH': 'Ohio',\n",
+    "    'TN': 'Tennessee',\n",
+    "    'FL': 'Florida',\n",
+    "    'PA': 'Pennsylvania',\n",
+    "    'CA': 'California',\n",
+    "    'TX': 'Texas',\n",
+    "    'VA': 'Virginia',\n",
+    "    'MO': 'Missouri',\n",
+    "    'GA': 'Georgia',\n",
+    "    'NY': 'New York',\n",
+    "    'CO': 'Colorado',\n",
+    "    'WA': 'Washington',\n",
+    "    'MN': 'Minnesota',\n",
+    "    'AL': 'Alabama',\n",
+    "    'IL': 'Illinois',\n",
+    "    'NC': 'North Carolina',\n",
+    "    'OK': 'Oklahoma',\n",
+    "    'MD': 'Maryland',\n",
+    "    'MI': 'Michigan',\n",
+    "    'LA': 'Louisiana',\n",
+    "    'SC': 'South Carolina',\n",
+    "    'nan': 'Unknown',\n",
```

Comment:
Are you depending on the places to exist for your Tableau plots? If not, this is fine

Reply:
That's the space I used to move to the next line. It's just the way I typed it. I tested it in Tableau and it works fine.